app/lib/efile/ny/it213.rb

```diff
@@ -131,8 +133,21 @@ def calculate_line_3
         @lines[:IT201_LINE_19].value <= cutoff_for_filing_status ? 1 : 2
       end
 
+      def calculate_line_5
+        count = 0
+        @intake.dependents.each do |dependent|
+          if dependent.relationship.present?
+            is_valid_relationship = %w[daughter stepchild foster_child grandchild sister nephew half_sister stepbrother son brother niece half_brother stepsister].include?(dependent.relationship.downcase)
+          end
+          if dependent.odc_qualifying == true && dependent.under_17? && is_valid_relationship
+            count += 1
+          end
+        end
```

Comment:
Looks like this is very similar logic as `eligible_for_child_tax_credit`, can that method be used here (might need some adjustments)?

Reply:
yes! i can reuse this method here.